packages/core/test/system.test.ts

```diff
@@ -273,9 +273,14 @@ describe('system', () => {
         src: createMachine({}),
         systemId: 'test'
       },
-      entry: assign(({ system }) => {
-        expect(system!.get('test')).toBeDefined();
-      })
```

Comment:
well, this is an ability that we'd "lose" if we gonna land this PR. I think it's fine though because this:
```ts
invoke: { src: logic, id: 'myId' },
entry: sendTo('myId', event)
```
is way more useful and probable than:
```ts
invoke: { src: logic, systemId: 'mySystemId' },
entry: sendTo(({ system }) => system.get('mySystemId'), event)
```

Note that this whole problem is **only** relevant when an entry action tries to grab the actor that is just about to be invoked by the very same state from the system. As showcased by the test adjustment - the actor becomes available in the system right after entry+invoke on a single state are resolved

Reply:
Yeah, that makes sense